backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/scheduling/policyunits/CpuPinningPolicyUnit.java

```diff
@@ -103,6 +103,7 @@ public List<VDS> filter(final SchedulingContext context,
                 }
                 break;
             case DEDICATED:
+            case ISOLATE_THREADS:
```

Comment:
I think that the code would be more readable if we dropped the `switch` and used a simple `if` instead. We already have `MANUAL` behaving like `RESIZE_AND_PIN` and now even the `DEDICATED` is actually the same as `ISOLATE_THREADS`. We can return all hosts for `NONE` and then check for exclusive pinning and at the end the "shared" pinning.